src/Plugin.InAppBilling/InAppBilling.ios.cs

```diff
@@ -350,92 +353,135 @@ public async override Task<InAppBillingPurchase> PurchaseAsync(string productId,
 
 		async Task<SKPaymentTransaction> PurchaseAsync(string productId, ItemType itemType, string applicationUserName, CancellationToken cancellationToken)
 		{
+            if (shouldConsoleLog)
+            {
+                paymentObserver.TurnOnConsoleLog();
```

Comment:
Enabling console logging in the paymentObserver. This is not ideal but I wanted to limit how much I changed that object.